cognite/client/config.py

```diff
@@ -96,6 +96,11 @@ def __init__(
                 from cognite.client.utils._auxiliary import _check_client_has_newest_major_version
 
                 _check_client_has_newest_major_version()
+        self._validate_config()
+
+    def _validate_config(self) -> None:
+        if not self.project:
+            raise ValueError(f"Invalid value for ClientConfig.project: <{self.project}>")
```

Comment:
wouldn't self.project be null here? 

okay the error will be. `Invalid value for ClientConfig.project: <null>`

👍